### DISCUSSION TITLE
Error on try to delete a user, prisma saying its related with other model, but the model do not exist because i have error when i try to create the model 'house'

### DISCUSSION THREAD

**Discussion by DarlonHenrique** [prisma/prisma#16697] [Q&A] [answered]
<img width="1680" alt="Screen Shot 2022-12-08 at 12 59 34 PM" src="https://user-images.githubusercontent.com/67025443/206496784-590f000b-4493-4cbb-84d7-1008e0f2e393.png">
<img width="1680" alt="Screen Shot 2022-12-08 at 12 52 11 PM" src="https://user-images.githubusercontent.com/67025443/206496842-1f4fe997-7a79-4dc2-8fc4-a145912e0a25.png">
<img width="1680" alt="Screen Shot 2022-12-08 at 12 51 45 PM" src="https://user-images.githubusercontent.com/67025443/206496921-8dfc59f2-8910-4694-ba82-e1173ff5eb1c.png">

**Comment by nurul3101** [comment, marked as answer]
Hey @DarlonHenrique 👋 
Can you please share your schema file? It would help in understanding the relations between User and House models.
And also, can you share the delete query which you are invoking?
Just to confirm, when do you get this error? While creating a house or while deleting a user?

**Reply by DarlonHenrique** [reply]
it's solved bro, I don't know why the house don't appear on insomnia, but I found it on prisma studio and delete successfully

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
